app/models/feed.rb

```diff
@@ -28,7 +28,11 @@ def from_redis(limit, max_id, since_id, min_id)
       unhydrated = redis.zrangebyscore(key, "(#{min_id}", '+inf', limit: [0, limit], with_scores: true).map(&:first).map(&:to_i)
     end
 
-    Status.where(id: unhydrated).cache_ids
+    Status.send(*status_scope).where(id: unhydrated).cache_ids
+  end
+
+  def status_scope
+    :all
```

Comment:
I think, if we are going to do something like this, it would be cleaner and more understandable to have something along the lines of “status_scope.where(...)” and have status_scope just be “Status”